firefox_ui_tests/remote/security/test_mixed_script_content_blocking.py

```diff
@@ -0,0 +1,116 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import By, Wait
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestMixedScriptContentBlocking(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+        self.url = 'https://mozqa.com/data/firefox/security/mixed_content_blocked/index.html'
+        self.test_elements = [
+            ('result1', 'Insecure script one'),
+            ('result2', 'Insecure script from iFrame'),
+            ('result3', 'Insecure plugin'),
+            ('result4', 'Insecure stylesheet'),
+        ]
+        self.locationbar = self.browser.navbar.locationbar
+
+    def tearDown(self):
+        try:
+            self.marionette.set_context("chrome")
+            self.marionette.execute_script("arguments[0].hidePopup();",
+                                           script_args=[self.locationbar.notification_popup])
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    def _expect_protection_status(self, enabled):
+
```

Comment:
nit: lets be consistent and lets get those empty lines removed from the beginning of a method.